src/EnergyPlus/PhaseChangeModeling/HysteresisModel.cc

```diff
@@ -165,7 +165,7 @@ namespace HysteresisPhaseChange {
                     phaseChangeState = PhaseChangeStates::FREEZING;
                     this->enthNew =
                         this->getEnthalpy(updatedTempTDT, this->peakTempFreezing, this->deltaTempFreezingLow, this->deltaTempFreezingHigh);
-                } else if (this->enthNew < this->enthalpyF && this->enthNew > this->enthalpyM) {
+                } else if (this->enthNew > this->enthalpyM && this->enthNew < this->enthalpyF) {
```

Comment:
Reordered some of the IF blocks so that Clang Format will stop trying to format them as template type argument specifiers.  Even Clang Format 17 is getting confused about it.

Reply:
There should just be parenthesis around each condition, that'd improve readability too. And I think clang format will stop messing up.

```suggestion
                } else if ((this->enthNew < this->enthalpyF) && (this->enthNew > this->enthalpyM)) {
```